README.md

```diff
@@ -206,15 +206,48 @@ Read the important message after you run `scxa-solrlcoud-bootstrap`:
 > takes a few extra minutes: the exception is thrown before the process has completed.
 > The best option is to manually build and supervise this step.
 >
-> Run the following command (don’t worry if the request returns a 500 error after a while):
+> Run the following commands one by one(don’t worry if the request returns a 500 error after a while):
 > 
 > `docker exec -i scxa-solrcloud-1 curl 'http://localhost:8983/solr/bioentities-v1/suggest?suggest.build=true&suggest.dictionary=propertySuggesterNoHighlight'`
+
+> `docker exec -i scxa-solrcloud-1 curl 'http://localhost:8983/solr/bioentities-v1/suggest?suggest.build=true&suggest.dictionary=bioentitySuggester'`
 >
 > While the command above is running, monitor the size of the suggester directory size:
 > 
 > `docker exec -it scxa-solrcloud-1 bash -c 'watch du -sc server/solr/bioentities-v1*/data/*'`
 > `docker exec -it scxa-solrcloud-2 bash -c 'watch du -sc server/solr/bioentities-v1*/data/*'`
 >
-> The suggester in all shards will be built when the propertySuggester directory size stabilises. 
-> Run the above procedure for `bioentitySuggester` as well.
- 
+> The suggester in all shards will be built when the propertySuggester directory size stabilises.
```

Comment:
We can generalise this comment for all suggesters then:
> Both suggesters in every shard will be built when the suggester directory size stabilises.

Reply:
Sure

Reply:
Fixed